src/fe-fuzz/Makefile.am

```diff
@@ -0,0 +1,25 @@
+bin_PROGRAMS = irssi-fuzz
+
+# Force link with clang++ for libfuzzer support
+CCLD=clang++ $(CXXFLAGS)
```

Comment:
Probably do a check if `clang++` is available if fuzzing is enabled in `configure.ac`?